src/python/WMCore/WMSpec/StdSpecs/ReReco.py

```diff
@@ -161,8 +161,6 @@ def __call__(self, workloadName, arguments):
                 skimConfig["SkimJobSplitArgs"]["files_per_job"] = int(arguments.get("SkimFilesPerJob%s" % skimIndex, 1))
             elif skimConfig["SkimJobSplitAlgo"] == "EventBased" or skimConfig["SkimJobSplitAlgo"] == "EventAwareLumiBased":
                 skimConfig["SkimJobSplitArgs"]["events_per_job"] = int(arguments.get("SkimEventsPerJob%s" % skimIndex, int((8.0 * 3600.0) / skimConfig["TimePerEvent"])))
-                if skimConfig["SkimJobSplitAlgo"] == "EventAwareLumiBased":
-                    skimConfig["SkimJobSplitAlgo"]["max_events_per_lumi"] = 20000
```

Comment:
Now I see why we were setting a default value during request creation. That's because we use the "splitting" section when retrieving the job splitting (at least via web UI) and only those attributes found are shown in the web UI. I'm updating this PR